src/components/Icon/Expensicons.js

```diff
@@ -0,0 +1,21 @@
+import ChatBubble from '../../../assets/images/chatbubble.svg';
+import Close from '../../../assets/images/close.svg';
+import MagnifyingGlass from '../../../assets/images/magnifyingglass.svg';
+import Paperclip from '../../../assets/images/paperclip.svg';
+import Pencil from '../../../assets/images/pencil.svg';
+import Pin from '../../../assets/images/pin.svg';
+import Plus from '../../../assets/images/plus.svg';
+import Send from '../../../assets/images/send.svg';
+import Users from '../../../assets/images/users.svg';
+
+export {
+    ChatBubble,
+    Close,
+    MagnifyingGlass,
+    Paperclip,
+    Pencil,
+    Pin,
+    Plus,
+    Send,
+    Users,
+};
```

Comment:
I don't love the idea of having this file collection, rather than simply importing each SVG asset to the file as and when it is required. What benifit do we get here with this extra step?

- Can import all icons in one import statement ➕ 
- If I want to find all usages of Send icon, I can't search by file name but need to search for the generic Send keyword ➖ 
- Adds an additional step to the process, which might not be obvious ➖ 
- If I browse icons and find `chatbubble.svg`, I need to check this file to confirm the export name ➖  (I know its just camelCase, but still)

Reply:
Yeah, the kicker is really that first point - because the collection of available icons is used by the `Icon` component itself. Would you feel better about it if the icons were exported from the `Icon` module itself? That's how I originally had it, and I think I actually like it better like so:

```javascript
import Icon, {MagnifyingGlass} from '../components/Icon';
```

Reply:
That way at least, you could search from all usages of the `Send` icon by looking for `import Icon, {.*Send.*}`

Reply:
My main issue is that we add an extra step in the add Icon process just to clean up our imports (which certainly looks nicer). 

I don't think this is a huge problem though.

Reply:
Lets keep it as is